src/data.rs

```diff
@@ -287,6 +304,7 @@ mod printing {
                 Visibility::Restricted(ref path) => {
                     tokens.append("pub");
                     tokens.append("(");
+                    tokens.append("in");
```

Comment:
This converts `pub(self)` to `pub(in self)` (and same for `super`) which technically has the same semantics but also isn't exactly equal to the input.